mava/networks/retention.py

```diff
@@ -0,0 +1,309 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Tuple
+
+import flax.linen as nn
+import jax
+import jax.numpy as jnp
+from chex import Array
+from omegaconf import DictConfig
+
+from mava.utils.sable_utils import PositionalEncoding
+
+
+class SimpleRetention(nn.Module):
```

Comment:
Can we rename this to just retention :pray:

Reply:
I think SimpleRetention is more informative because we have the file called retention... (but we can chat about it offline)